### ISSUE TITLE
No bug in fnc def?

### ISSUE THREAD

**Issue by Alexey-T** [pyscripter/python4delphi#205]
function PyUnicode_FromAnsiString(const Str: AnsiString): PPyObject;
    function PyUnicode_FromString(const Str : string): PPyObject;

For Lazarus I have string==ansistring. For D7 too. So its bug?

**Comment by pyscripter** [comment]
What exactly is the bug?
We have 
```pascal
function TPythonEngine.PyUnicode_FromWideString( const AString : UnicodeString) : PPyObject;
function PyUnicode_FromAnsiString(const Str: AnsiString): PPyObject;
function PyUnicode_FromString(const Str : string): PPyObject;
```
The last one is to to use with the native string type (Ansistring or UnicodeString), without having to typecast.

**Comment by pyscripter** [comment]
Would it better to have?
```pascal
function PyUnicode_FromString(const Str : AnsiString): PPyObject; overload;
function PyUnicode_FromString(const Str : UnicodeString): PPyObject; overload;
```

**Comment by Alexey-T** [comment]
Yes, last variant is better (old variant is OK for FPC too, though)

**Comment by pyscripter** [comment]
Now we have:
```
    {Imported functions - Assume/work with utf8}
    PyUnicode_FromString:function (s:PAnsiChar):PPyObject; cdecl;
    PyUnicode_AsUTF8:function (unicode: PPyObject):PAnsiChar; cdecl;
```
and
```
    { String conversion }
    function PyUnicodeFromString(const AString : UnicodeString) : PPyObject; overload;
    function PyUnicodeFromString(const AString: AnsiString): PPyObject; overload;
    function PyUnicodeAsString( obj : PPyObject ) : UnicodeString;
```
PyUnicode_AsUTF8 is only available from python 3.3 onwards and I dropped support for python 3.2
